### ISSUE TITLE
Issues with Shaders

### ISSUE THREAD

**Issue by valtheriault** [WesCook/Homestead#105]
Firstly, this modpack is by far my favourite on the atl laucher, I love how challenging it is!

I realize this isn't an issue with your modpack per say and more with specific mods being incompatible with shaders, but I was wondering if you could provide direction since I'm not super familiar with modifying minecraft. 

I've already gone into astral sorcery's config and enabled weakSkyRenders and set "Use Shaders" in the dsurround config to false, but I'm getting crashes while playing the game still as well as flickering in the sky at night. I have my crash report attached here, if you're unable to help, no problem! :) Thank you for creating this modpack!

[crash-2020-05-26_18.43.50-client.txt](https://github.com/WesCook/Homestead/files/4685411/crash-2020-05-26_18.43.50-client.txt)

**Comment by WesCook** [comment]
I haven't done any testing with shaders, but I'd try disabling these mods first:

* Enhanced Visuals
* Mo' Bends
* Blur

Each is visual only but plays with the rendering system.

**Comment by valtheriault** [comment]
Hi! Thanks for the advice! I was able to solve both the night sky flicker and the crashing! 

I couldn't use an already existing save, which was my first issue. I either corrupted it while testing shaders or something of the like because it crashes pretty consistently now. (I am now realizing after looking at another issue in here that I'm having that exact same internal server error on that world save - it started happening recently though after 50 days in so its likely my fault)

In case anyone else wants to use shaders with your awesome modpack:
Using ATL Launcher (probably similar for other launchers)
1. Create a new instance of homestead and do NOT install Enhanced Visuals or Optifine. All other optional mods are fine, I have them all including Mo'Bends and Blur.
2. Manually download Optifine from their website for version 1.10.2 of minecraft and place it in the mod directory.
3. Go to the config folder of your instance and modify astralsorcery.cfg with something like notepad++ and look for "weakSkyRenders", add a 0 in between the < > (like this: < 0 > )
4. Downloade the shader you want, put it in the shaderspack folder of your instance.
5. Start up the instance and immediately go to your video settings and select that shader, wait for it to apply (it can take a few minutes and it will appear to freeze but it's not frozen).
6. Create a NEW game. I don't think existing worlds work, at least it didn't for me.

I'll keep playing around to make sure it doesn't randomly start crashing, but the fact that the sky doesn't flicker anymore and I was able to play through a day cycle is a good start!
